docs/source/command_line.rst

```diff
@@ -290,6 +290,13 @@ Here are some more useful flags:
   of the parameters or the return type). ``unannotated`` option is
   interchangeable with ``--disallow-untyped-defs``.
 
+  ``expr`` disallows all expressions in the module that have type ``Any``.
+  If an expression of type ``Any`` appears anywhere in the module
+  mypy will output an error unless the expression is immediately
+  used as an argument to ``cast`` or assigned to a variable with an
+  explicit type annotation. Note that declaring a variable of type ``Any``
+  or casting to type ``Any`` is not allowed.
```

Comment:
No, `cast(Any, foo)` and `x: Any = foo` are not allowed (we discussed it [here](https://github.com/python/mypy/pull/3519#discussion_r121779999) and [here](https://github.com/python/mypy/pull/3519#discussion_r121780369)).

What do you think of the wording?